Lib/test/test_subprocess.py

```diff
@@ -1709,7 +1732,7 @@ def raise_it():
                                  preexec_fn=raise_it)
         except subprocess.SubprocessError as e:
             self.assertTrue(
-                    subprocess._posixsubprocess,
+                    mock_submod,
```

Comment:
given the skipIf I think this could be left as subprocess._posixsubprocess.  `mock_submod` would be better named `platform_specific_extension_module` at which point i'd object less to it being used here.